src/hubbleds/utils.py

```diff
@@ -126,3 +133,61 @@ def data_summary_for_component(data, component_id):
         summary[f"{percent}%"] = (bottom, top)
 
     return summary
+
+
+M = TypeVar("M", bound=BaseModel)
+def models_to_glue_data(items: List[M],
```

Comment:
I appreciate the generics, but is it possible to want to convert a list of inconsistent `BaseModel` subclasses to a glue `Data` object, or do we always expect them to be the same?

Reply:
I was imagining that all of the items would be of the same type - glue wants every component to have the same shape, so I think we want every model instance to have the same structure here. Technically this setup still could have an issue where the first item is a subclass of `M` with additional model fields, but I'm not sure if there's a way around that.